src/Shapefile.jl

```diff
@@ -133,17 +297,21 @@ mutable struct Handle{T<:Union{<:GeoInterface.AbstractGeometry,Missing}}
     mrange::Interval
     shapes::Vector{T}
 end
-
 function Handle(path::AbstractString, index=nothing)
     open(path) do io
         read(io, Handle, index)
     end
 end
-
-
+function Handle(path::AbstractString, indexpath::AbstractString)
```

Comment:
These are changes unrelated to the docs, same for most of the changes below here.

Reply:
Ok somehow an unrelated PR snuck in